CHANGES.md

```diff
@@ -4,6 +4,16 @@
   `identity` has been removed.
 
   _Migration guide:_ use `Universum.id` instead.
+* [#178](https://github.com/serokell/universum/issues/178):
+  `getArgs` has been removed.
+
+  _Migration guide:_ use `liftIO` directly with `System.Environment.getArgs`
+  from base.
+* [#179](https://github.com/serokell/universum/issues/179):
```

Comment:
Nitpick: it's in the commit which removes `getArgs`, can you move it to the commit where this change (removal of `getContents` and `interact`) is actually done?